aspnetcore/performance/response-compression/sample/ResponseCompressionSample.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
I thought ` ToolsVersion="15.0"` was going away. I'll mirror what's in the samples you linked.